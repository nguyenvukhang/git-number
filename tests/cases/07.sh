# use range (add 1 4-6)

init 7
_gitnu status
_gitnu add 1 4-6
save gitnu status
