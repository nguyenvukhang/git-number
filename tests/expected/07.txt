On branch main

No commits yet

Changes to be committed:
1	[32mnew file:   file_0001[m
2	[32mnew file:   file_0004[m
3	[32mnew file:   file_0005[m
4	[32mnew file:   file_0006[m

Untracked files:
5	[31mfile_0002[m
6	[31mfile_0003[m
7	[31mfile_0007[m

